### PR TITLE
Rename ssr

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Build's going to fail because it's not yet published.

File: optimizer-express/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ An amphtml link will be added to the server-side-rendered version:
 
 ## Usage
 
-The AMP SSR middleware can be used like any other express middleware.
+The AMP Optimize Middleware can be used like any other express middleware.
```
Comment (andreban) [inline optimizer-express/README.md]: s/AMP Optimize Middleware/AMP Optimizer Middleware

File: optimizer-express/lib/AmpOptimizeMiddleware.js [hunk 2]
```diff
@@ -18,57 +18,57 @@
 /**
  * The transform middleware replaces the `res.write` method so that, instead of sending
  * the content to the network, it is accumulated in a buffer. `res.end` is also replaced
- * so that, when it is invoked, the buffered response is transformed with AMP-SSR and sent
+ * so that, when it is invoked, the buffered response is transformed with AMP-Optimize and sent
```
Comment (andreban) [inline optimizer-express/lib/AmpOptimizeMiddleware.js]: s/AMP-Optimize/AMP-Optimizer

File: optimizer-express/lib/AmpOptimizeMiddleware.js [hunk 3]
```diff
@@ -18,57 +18,57 @@
...
 
 const DEFAULT_URL_MAPPING = new UrlMapping('amp');
 
-class AmpSsrMiddleware {
+class AmpOptimizerMiddleware {
 
   /**
    * @function runtimeVersion A function used to provide the runtimeVersion when applying the
-   * SSR transformations.
+   * optimize transformations.
```
Comment (andreban) [inline optimizer-express/lib/AmpOptimizeMiddleware.js]: s/optimize/optimizer

File: optimizer-express/lib/AmpOptimizeMiddleware.js [hunk 4]
```diff
@@ -113,15 +113,15 @@ class AmpSsrMiddleware {
...
+          .then(version => optimizer.transformHtml(body, {
             ampUrl: linkRelAmpHtmlUrl,
             ampRuntimeVersion: version
           }))
           .then(transformedBody => {
             res.status(200).send(transformedBody);
           })
           .catch(err => {
-            console.error('Error applying AMP SSR transformations. Sending original page', err);
+            console.error('Error applying AMP Optimize transformations. Sending original page', err);
```
Comment (andreban) [inline optimizer-express/lib/AmpOptimizeMiddleware.js]: s/Optimize/Optimizer

File: optimizer-express/spec/lib/AmpSsrMiddlewareSpec.js [hunk 5]
```diff
@@ -92,9 +92,9 @@ describe('Express Middleware', () => {
       transformHtml: () => Promise.reject('error')
     };
 
-    const middleware = AmpSsrMiddleware.create({ampSsr: transformer});
+    const middleware = AmpOptimizerMiddleware.create({ampOptimizer: transformer});
 
-    it('Sends the original content when SSR fails', () => {
+    it('Sends the original content when Optimize fails', () => {
```
Comment (andreban) [inline optimizer-express/spec/lib/AmpSsrMiddlewareSpec.js]: s/Optimize/Optimizer

File: optimizer/README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-`amp-toolbox-ssr` is a library for server-side-rendering AMP pages. This makes it possible to optimize
+`amp-toolbox-optimizer` is a library for server-side-rendering AMP pages. This makes it possible to optimizer
 page loading times when serving AMP pages from a non-cache domain.
 
 **Warning:** server-side rendered AMP files are no longer valid AMPHTML. Hence, serve-side rendered AMP pages should link to their [valid AMP counterpart](https://www.ampproject.org/docs/guides/discovery).
```
Comment (andreban) [inline optimizer/README.md]: s/optimizer/optimize - In this case, optimize is not the project name, but part of the text.

File: optimizer/README.md [hunk 7]
```diff
@@ -28,18 +28,18 @@ The [Google AMP Cache](https://developers.google.com/amp/cache/overview#cache-op
 
 ## Usage
 
-You can find a sample implementation [here](demo/simple/). If you're using the express middleware, please use the [AMP SSR middleware](../ssr-express).
+You can find a sample implementation [here](demo/simple/). If you're using the express middleware, please use the [AMP Optimize Middleware](../optimizer-express).
```
Comment (andreban) [inline optimizer/README.md]: s/Optimize/Optimizer

File: optimizer/demo/cheerio/index.js [hunk 8]
```diff
@@ -54,8 +54,8 @@ class CheerioTransformer {
    */
   transform(tree, params) {
     const $ = cheerio.load(tree.root.children);
-    // Prepends "SSR" to the <title>
-    $('title').text('SSR: ' + $('title').text());
+    // Prepends "Optimized" to the <title>
+    $('title').text('Optimize: ' + $('title').text());
```
Comment (andreban) [inline optimizer/demo/cheerio/index.js]: s/Optimize/Optimized

File: optimizer-express/README.md (hunk 1 above)
Comment (sebastianbenz) [reply optimizer-express/README.md]: done

File: optimizer-express/lib/AmpOptimizeMiddleware.js (hunk 2 above)
Comment (sebastianbenz) [reply optimizer-express/lib/AmpOptimizeMiddleware.js]: done

File: optimizer-express/lib/AmpOptimizeMiddleware.js (hunk 3 above)
Comment (sebastianbenz) [reply optimizer-express/lib/AmpOptimizeMiddleware.js]: done

File: optimizer-express/lib/AmpOptimizeMiddleware.js (hunk 4 above)
Comment (sebastianbenz) [reply optimizer-express/lib/AmpOptimizeMiddleware.js]: done

File: optimizer-express/spec/lib/AmpSsrMiddlewareSpec.js (hunk 5 above)
Comment (sebastianbenz) [reply optimizer-express/spec/lib/AmpSsrMiddlewareSpec.js]: done

File: optimizer/README.md (hunk 6 above)
Comment (sebastianbenz) [reply optimizer/README.md]: done

File: optimizer/README.md (hunk 7 above)
Comment (sebastianbenz) [reply optimizer/README.md]: done

File: optimizer/demo/cheerio/index.js (hunk 8 above)
Comment (sebastianbenz) [reply optimizer/demo/cheerio/index.js]: done

**Pull request merged**